sql/hive/pom.xml

```diff
@@ -56,6 +56,12 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-sql_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
+      <exclusions>
```

Comment:
If we don't exclude it, then the hive module will depend on a shaded parquet lib, right?

Assuming it's not excluded, then the hive module depends on both the shaded parquet, and the standard parquet that is brought by the hive lib. Does it cause any problem?

Reply:
Only one problem: https://github.com/apache/spark/pull/24371#pullrequestreview-227243175